keanu-project/src/main/java/io/improbable/keanu/util/csv/pojo/PublicFieldMatcher.java

```diff
@@ -39,7 +48,7 @@
             try {
                 matchingField.set(target, convertedValue);
             } catch (IllegalAccessException e) {
-                throw new RuntimeException(e);
+                log.warn(Arrays.toString(e.getStackTrace()));
```

Comment:
Can we recover from this?

Reply:
Nope, good spot. I've called an `IllegalArgumentException`